### PR TITLE
changed some text

### PR DESCRIPTION
**Pull request by NielsMartensGitH**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -25,7 +24,7 @@ app.use(express.urlencoded({extended: false}));
 app.use('/api/products', productRoute);
 
 app.get('/', (req, res) => {
-    res.send('Hello Node API !!!');
```
Comment (NielsMartensGitH) [inline server.js:28]: test

**Review by NielsMartensGitH — COMMENTED** [review]
ok

**Review by NielsMartensGitH — COMMENTED** [review]
reviewed

**Review by NielsMartensGitH — COMMENTED** [review]
ok

File: server.js
Comment (NielsMartensGitH) [inline server.js:1]: test

**Review by NielsMartensGitH — COMMENTED** [review]
test

File: server.js
Comment (NielsMartensGitH) [inline server.js:1]: start

**Review by NielsMartensGitH — COMMENTED** [review]
finished review

**Comment by NielsMartensGitH** [comment]
test

File: server.js
Comment (NielsMartensMediahuis) [inline server.js:1]: good

**Review by NielsMartensMediahuis — APPROVED** [review]

**Pull request merged**